python/foglamp/services/core/service_registry/monitor.py

```diff
@@ -44,8 +44,11 @@ def __init__(self):
         self._ping_timeout = None  # type: int
         """Timeout for a response from any given micro-service"""
 
+    async def _sleep(self, sleep_time):
```

Comment:
Note that I put a wrapper around asyncio.sleep. The code runs an infinite loop in which asyncio.sleep is called routinely. I wanted to be able to use the sleep as a safe exit point (mock sleep to throw an exception when I desire it to do so). I also wanted to eliminate sleep time so that the unit test executes quickly. @MarkRiddoch  please let me know if this is okay. It is strictly a change to accommodate unit tests.

Reply:
Yes certainly, in this case we are not creating a different code flow for test, so I think it is fine.